utils/update-db.js

```diff
@@ -296,8 +300,8 @@ class KanjiDictParser extends Writable {
     let hasB = false;
     for (const ref of refs) {
       if (ref.length && SUPPORTED_REF_TYPES.has(ref[0])) {
-        // Special case Dx types since we only support DK types
-        if (ref[0] === 'D' && ref.slice(0, 2) !== 'DK') {
+        // Special case Dx types since we only support DK and DN types
+        if (ref[0] === 'D' && !['K', 'N'].includes(ref[1])) {
```

Comment:
Is there a good reason why this checks character by character? Seems a little funky!

Reply:
It was because it was hard to tell when the ref letter code stopped and when the content it described began. (DK46 and I12 for example)

I still think a generic solution is better so I changed the code to check if each ref was prefixed by a supported ref.

Might be good to factor the logic into helper variables/functions though.